### DISCUSSION TITLE
[Bug]:X-Frame-Options Cross-Domain Issue

### DISCUSSION THREAD

**Discussion by liuxiawei** [django-wiki/django-wiki#1291] [Bug] [answered]
### Describe the bug

In class MergeView, It should add `@method_decorator(xframe_options_sameorigin)` in line 913
![image](https://github.com/django-wiki/django-wiki/assets/5093541/637bca17-1b7b-4bfc-833f-54c76e418e2a)
Or it would be error while merge article

### Steps to reproduce.

1. Go to 'Article history'
2. Click on 'Merge Selected with current ...'

### Python Version

3.10.8

### Django Version

4.2

### Extra context

Firefox

**Comment by oscarmcm** [comment, marked as answer]
Scaled to #1293 and fixed in #1294
